scene/3d/vehicle_body_3d.cpp

```diff
@@ -948,13 +924,6 @@ void VehicleBody3D::_bind_methods() {
 }
 
 VehicleBody3D::VehicleBody3D() {
-	m_pitchControl = 0;
-	m_currentVehicleSpeedKmHour = real_t(0.);
-	m_steeringValue = real_t(0.);
-
-	engine_force = 0;
-	brake = 0;
-
 	state = nullptr;
 	ccd = false;
```

Comment:
Could be removed, assuming that they're the same default values as in PhysicsBody3D.